src/main.jsx

```diff
@@ -98,19 +103,27 @@ ReactDOM.createRoot(document.getElementById("root")).render(
           }
         />
       </Routes>
-      <NavTray
-        isLoggedIn={false}
-        navLinks={[
-          {
-            role: "login",
-            to: "/login",
-          },
-          {
-            role: "home",
-            to: "/",
-          },
-        ]}
-      />
+      {isLoggedIn && (
+        <NavTray
+          navLinks={[
+            {
+              role: "profile",
+              to: "/profile",
+              Icon: ProfileIcon,
+            },
+            {
+              role: "home",
+              to: "/",
+              Icon: HomeIcon,
+            },
+            {
+              to: "/account",
```

Comment:
I think this is a typo should be "/notifications"

Reply:
Yes, but there is not any notification page yet, to avoid possible errors I add existing path.